app/views/aspect_memberships/_aspect_membership_dropdown.haml

```diff
@@ -9,7 +9,7 @@
         = t('shared.aspect_dropdown.toggle', :count => selected_aspects.size)
     %span.caret
 
-  %ul.dropdown-menu{:class => ["pull-#{hang}", defined?(dropdown_class) && dropdown_class], :unSelectable => 'on', 'data-person_id' => (person.id if defined?(person) && person), 'data-service_uid' => (service_uid if defined?(service_uid)), 'data-person-short-name' => (person.first_name if defined?(person) && person)}
+  %ul.dropdown-menu{:class => ["pull-#{hang}", "aspect_membership"], :unSelectable => 'on', 'data-person_id' => (person.id if defined?(person) && person), 'data-service_uid' => (service_uid if defined?(service_uid)), 'data-person-short-name' => (person.first_name if defined?(person) && person)}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.